src/lib.rs

```diff
@@ -1,3 +1,8 @@
+/// # Fantom Libtransport-tcp
+///
+/// This library is an asynchronous TCP implementation of the Transport trait in libtransport
```

Comment:
This library is TCP implementation of the Transport trait in libtransport in async/.await framework of the Rust.